Constants and routing fixed

pranjal-garg opened the pull request:
Optimized code by organizing strings, image, and icon paths into the constants folder and implemented named routing across the app.


https://github.com/user-attachments/assets/a6657e33-e821-4710-bee2-5b505b000a18

archeeetah commented on lib/utilities/constants/strings.dart:
Name the Text file as Text_string not string and change the class name as well

pranjal-garg replied:
Will fix the naming convention

archeeetah commented on lib/utilities/constants/icons_const.dart:
Move asset-based icon URLs (e.g., .png, .svg) to the image constants file, and keep Flutter vector icons (e.g., Icons.) in a separate icon constants file

pranjal-garg replied:
Will move asset icons to image constants file

pranjal-garg commented:
I've addressed the review feedback:
- Renamed KStrings.dart to Text_string.dart
- Moved asset-based icons to images_const.dart  
- Moved vector-based icons to icons_const.dart
- Added remaining registration screen strings to Text_string constants

Tested the complete app flow and confirmed all features and navigation are functioning correctly.

pranjal-garg commented:
Implemented the changes told in the meeting:
- Moved all screen colors to themes.dart
- Moved all screen TextStyles to themes.dart  
- Moved all component colors to themes.dart
- Moved all component TextStyles to themes.dart
- Renamed Text_strings class to KText

All changes have been tested and are working correctly.

archeeetah commented on lib/app/modules/Settings/views/settings_view.dart:
```diff
@@ -28,7 +25,7 @@ class SettingsView extends GetView<SettingsController> {
             decoration: BoxDecoration(
               gradient: LinearGradient(
-                colors: [theme.colorLevel1, theme.colorLevel2],
+                colors: [theme.darkBlue, theme.darkTeal],
```
Define color using Get.theme

archeeetah commented on lib/app/modules/Settings/views/settings_view.dart:
```diff
@@ -43,7 +40,7 @@ class SettingsView extends GetView<SettingsController> {
                     height: screenHeight * 0.2,
                     decoration: BoxDecoration(
-                      color: theme.colorLevel2,
+                      color: theme.darkTeal,
```
check wherever color is defined like this and correct it unless you have defined theme = Get.theme

archeeetah commented on lib/app/modules/meetings/components/DatePickerField.dart:
```diff
@@ -42,7 +44,7 @@ class DatePickerField extends StatelessWidget {
         controller: TextEditingController(text: '${selectedDate.value.year}-${selectedDate.value.month}-${selectedDate.value.day}'),
 
-          style: TextStyle(color: Colors.grey[700]),),
+          style: Get.theme.kInputFieldTextStyle2,),
```
check if you have defined the name correctly in themes.dart

archeeetah replied:
check if the color is same as defined in themes.dart

archeeetah commented on lib/app/modules/meetings/components/Dropdown.dart:
```diff
@@ -24,9 +26,9 @@ class CustomDropdownButton extends StatelessWidget {
             decoration: InputDecoration(
               hintText: hintText,
-              hintStyle: TextStyle(color: Colors.grey[700]),
+              hintStyle: Get.theme.kInputFieldTextStyle2,
```
check color here as well

archeeetah commented on lib/app/modules/meetings/components/TimePickerField.dart:
```diff
@@ -26,15 +29,15 @@ class TimePickerField extends StatelessWidget {
               decoration: InputDecoration(
                 hintText: hintText,
-                hintStyle: TextStyle(color: Colors.grey[700], fontWeight: FontWeight.w300),
+                hintStyle: Get.theme.kHintTextStyle2,
```
use copyWith if the attributes are not same as defined in themes

archeeetah commented on lib/app/modules/profileEdit/views/profile_edit_view.dart:
define colors using Get.theme

archeeetah reviewed (changes requested):
Once check if all the predefined colors are correct and the flow

pranjal-garg commented:
Addressed review feedback:
- Used Get.theme instead of direct theme reference for colors and styles
- `TextStyle get kInputFieldTextStyle2 => TextStyle(color: darkGrey);` defined at themes.dart:73
- `Color get darkGrey => Colors.grey[700] ?? Colors.grey;` defined at themes.dart:28
- `TextStyle get kHintTextStyle2 => TextStyle(color: darkGrey, fontWeight: FontWeight.w300);` defined at themes.dart:88

pranjal-garg commented:
Verified predefined colors and application flow - demo attached


https://github.com/user-attachments/assets/64403b7e-da5e-4b31-881f-368782eb0dbc
